src/app/layout.tsx

```diff
@@ -34,15 +34,15 @@ export default function RootLayout({
         />
         <meta
           property="og:image:alt"
-          content="Azure Resource Naming Convention Periodic Table"
+          content="The Azure Periodic Table"
```

Comment:
I like "The" because it makes it sound authoritative like The place to be for the information you need.  I'd like to keep that for now, unless you strongly object.